conda-forge.yml

```diff
@@ -4,3 +4,8 @@ travis:
 appveyor:
   secure:
     BINSTAR_TOKEN: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+channels:
+  sources:
+  - [conda-forge]
```

Comment:
Raised this issue upstream ( https://github.com/conda-forge/conda-smithy/issues/306 ).